### PR TITLE
chore: Add `README.md`

### PR DESCRIPTION
**Pull request by vrongmeal**

### CODE REVIEW

**Comment by infiniteregrets** [comment]
I think discord link and docs.rs should be a badge on top for quick access!

**Comment by vrongmeal** [comment]
> I think discord link and docs.rs should be a badge on top for quick access!

Ah yes, thanks! I'll add badges as well :)

File: README.md [hunk 1]
```diff
@@ -1 +1,99 @@
...
+> [!TIP]
+> You might want to update the basin name in the example before running since
+> basin names are globally unique and each example uses the same basin name
+> (`"my-basin"`).
+
+## SDK Docs and Reference
+
+Head over to [docs.rs](https://docs.rs/streamstore/latest/streamstore/) for
+detailed documentation and crate reference.
+
```
Comment (infiniteregrets) [inline README.md:77]: is this needed if badge is there

Comment (infiniteregrets) [reply README.md:77]: cool

**Review by infiniteregrets — APPROVED** [review]
🫨

**Pull request merged**